client/src/App.js

```diff
@@ -132,21 +132,23 @@ class App extends Component {
   getDriverPost = event => {
     // event.preventDefault();
     API.getDriverPost(this.state.id)
-          .then(results => {
-           console.log(results);
-            this.setState({ results: results.data })})
-          .catch(err => console.log(err));
-         
+      .then(results => {
+        console.log(results);
+        this.setState({ results: results.data })
+      })
+      .catch(err => console.log(err));
+
   }
 
   getRiderPost = event => {
     // event.preventDefault();
     API.getRiderPost(this.state.id)
-          .then(results => {
-           console.log(results);
-            this.setState({ riderPost: results.data })})
-          .catch(err => console.log(err));
-         
+      .then(results => {
+        console.log(results);
+        this.setState({ results: results.data })
```

Comment:
Should be kept as `this.setState({ riderPost: results.data })`. Dashboard doesn't work otherwise